chef-utils/lib/chef-utils/dsl/windows.rb

```diff
@@ -56,17 +59,19 @@ def windows_server?(node = __getnode)
 
       # Determine the current Windows NT version. The NT version often differs from the marketing version, but offers a good way to find desktop and server releases that are based on the same codebase. IE: NT 6.3 is Windows 8.1 and Windows 2012 R2.
       #
-      # @param [Chef::Node] node
+      # @param [Chef::Node] node the node to check
+      # @since 15.8
       #
       # @return [ChefUtils::VersionString]
       #
       def windows_nt_version(node = __getnode)
         ChefUtils::VersionString.new(node["os_version"])
       end
 
-      # Determine the installed version of PowerShell
+      # Determine the installed version of PowerShell.
       #
-      # @param [Chef::Node] node
+      # @param [Chef::Node] node the node to check
```

Comment:
The text `node the node to check` appears in this PR 73 times. It should be search-and-replaced to `the node to check`, probably?

Reply:
The first part is the param name. It's the format the vscode yard plugin and a few yard example pages I found use. Seems really strange, but 🤷‍♂